api4/saml.go

```diff
@@ -28,7 +28,7 @@ func (api *API) InitSaml() {
 
 	api.BaseRoutes.SAML.Handle("/certificate/status", api.APISessionRequired(getSamlCertificateStatus)).Methods("GET")
 
-	api.BaseRoutes.SAML.Handle("/metadatafromidp", api.APIHandler(getSamlMetadataFromIdp)).Methods("POST")
+	api.BaseRoutes.SAML.Handle("/metadatafromidp", api.APISessionRequired(getSamlMetadataFromIdp)).Methods("POST")
```

Comment:
why did this change from `APIHandler` to `APISessionRequired`?

Reply:
Same as in previous comment.